lighthouse-plugin-publisher-ads/messages/en-US.json

```diff
@@ -201,6 +201,11 @@
     "AREA_LARGER_THAN_VIEWPORT": "Calculated ad area is larger than viewport",
     "VIEWPORT_AREA_ZERO": "Viewport area is zero"
   },
+  "WARNINGS": {
+    "NO_ADS": "No ads were requested when fetching this page.",
+    "NO_AD_RENDERED": "No ads were rendered when fetching this page.",
```

Comment:
Nit: is "fetching" the correct verb to use on line 206? (I agree it is correct on line 205.)

Reply:
Agreed, thought it was awkward to use rendered twice though. WDYT about "No ads were painted when rendering this page"?

Reply:
Yeah, that works. Best I could come up with was "No ads were rendered when rendering this page" but that's somewhat repetitive and repetitive.